v3/src/data-model/data-set.ts

```diff
@@ -72,6 +72,14 @@ export interface IAddCaseOptions {
   // id(s) of case(s) before which to insert new cases
   // if not specified, new cases are appended
   before?: string | string[];
+  after?: string | string[];
```

Comment:
Can both 'before' and 'after' be specified? And what does it mean for there to be an array of ids?